packages/deployments/contracts/contracts/core/connext/libraries/AssetLogic.sol

```diff
@@ -307,8 +316,7 @@ library AssetLogic {
       IStableSwap pool = s.adoptedToLocalExternalPools[_key];
       address poolAddress = address(pool);
 
-      // NOTE: We revert here if the external stableswap pool doesn't exist.
-      if (poolAddress == address(0)) revert AssetLogic__swapAsset_externalStableSwapPoolDoesNotExist();
+      // NOTE: This call will revert if the external stableswap pool doesn't exist.
 
```

Comment:
The zero address check is missing. Kindly correct

Reply:
Calling non existing contracts is always tricky, but is solved by solidity: [solidity docs]:(https://docs.soliditylang.org/en/v0.8.17/control-structures.html#external-function-calls)
`
Due to the fact that the EVM considers a call to a non-existing contract to always succeed, Solidity uses the extcodesize opcode to check that the contract that is about to be called actually exists (it contains code) and causes an exception if it does not. This check is skipped if the return data will be decoded after the call and thus the ABI decoder will catch the case of a non-existing contract.`
Thus the 0 check isn't needed